osu.NET/Endpoints/Users.cs

```diff
@@ -249,4 +249,32 @@ public async Task<ApiResult<User[]>> LookupUsersAsync(int[] ids, bool? excludeBo
       .. ids.Select(x => ("ids[]", x)),
       ("exclude_bots", excludeBots)
     ], json => json["users"]);
+
+  /// <summary>
+  /// Returns all users with the specified IDs, up to 50. If a user ID could not be found, it is skipped.
+  /// <br/><br/>
+  /// This endpoint does not include any statistics. If you need this information,
+  /// use <see cref="GetUsersAsync(int[], bool?, CancellationToken?)"/> with a higher ratelimit cost.
+  /// <br/><br/>
+  /// API notes:
+  /// <list type="bullet">
+  /// <item>Includes <see cref="User.Country"/></item>
+  /// <item>Includes <see cref="User.Cover"/></item>
+  /// <item>Includes <see cref="User.Groups"/></item>
+  /// <item>Does *not* include <see cref="User.Statistics"/> or <see cref="User.RulesetStatistics"/></item>
+  /// </list>
+  /// API docs:<br/>
+  /// <a href="https://osu.ppy.sh/docs/index.html#get-users"/>
+  /// </summary>
+  /// <param name="ids">The user IDs.</param>
+  /// <param name="excludeBots">Optional. Bool whether bot users should be ignored.</param>
+  /// <param name="cancellationToken">Optional. The cancellation token for aborting the request.</param>
+  /// <returns>The users with the specified IDs.</returns>
+  [CanReturnApiError()]
+  public async Task<ApiResult<User[]>> LookupUsersAsync(string[] ids, bool? excludeBots = null, CancellationToken? cancellationToken = null)
+    => await GetAsync<User[]>($"users/lookup", cancellationToken,
+    [
+      .. ids.Select(x => ("ids[]", Uri.EscapeDataString(x))),
```

Comment:
The escaping is not necessary, `OsuApiClient.BuildRequestUrl` takes care of that. Although, feel free to place `HttpUtility.UrlEncode` with `Uri.EscapeDataString` there, as only latter is RFC 3986 conform.

https://github.com/minisbett/osu.NET/blob/2132d8188789d8c52967d9596d400e168e611c1b/osu.NET/OsuApiClient.cs#L174

Furthermore, if you take a closer look at osu-web, you can see that - similar to the v1 API - it uses an `@` prefix to distinguish number-only usernames from ids. In theory, you can even specify both usernames and IDs, but I believe it is fine to limit it to target one of those at a time.

```suggestion
      .. ids.Select(x => ("ids[]", $"@{x}")),
```